sourcegraph/sourcegraph
# Packages: hide git history panel

Fixes https://github.com/sourcegraph/sourcegraph/issues/37769

## Test plan

Manually tested with

```
❯ SOURCEGRAPHDOTCOM_MODE=true SOURCEGRAPH_API_URL=https://sourcegraph.com sg start web-standalone
```

Visit https://sourcegraph.test:3443/go/atomicgo.dev/cursor@0b3edc2989ae879a7d31344c4dd07d844551f79a/-/blob/cursor.go?L15:6-15:15#tab=history

* Confirm that the toggle button is disabled
* Confirm that the "History" tab is visible but has the message "Git history is not available when browsing packages"
